share/picongpu/examples/Bunch/include/picongpu/param/fieldBackground.param

```diff
@@ -46,7 +52,11 @@ namespace picongpu
          * Note: No PMACC_ALIGN(...) used, since this *additional* memory alignment would require
          *       roughly float_64 the number of registers in the corresponding kernel on the device.
          */
+#ifdef TWTSFAST
+        const templates : twtsfast::Efield twtsFieldE;
+#else
```

Comment:
Was `templates :` supposed to be `templates::` ? Same also lower in this file

Reply:
You are right thanks for the catch.